Automattic/wp-calypso
# Feature-example component: lower z-index not to collide with floating…

Lower z-index in Feature-example component to not collide with floating inline help button:

Fixes https://github.com/Automattic/wp-calypso/issues/32985

#### Before 

<img width="466" alt="image" src="https://user-images.githubusercontent.com/87168/57849975-74a99e00-77e5-11e9-85aa-bff3198fbe47.png">


#### After

<img width="547" alt="image" src="https://user-images.githubusercontent.com/87168/57846931-f1387e80-77dd-11e9-9b48-484b2729b4b9.png">


#### Changes proposed in this Pull Request

* Change FeatureExample component's z-index

#### Testing instructions
- Have a Jetpack site.
- Open `http://calypso.localhost:3000/plans/my-plan/:site?thank-you`
- scroll the page so that help button and faded gradient overlay
- Observe that gradient stays behind the floating help button
- Test both desktop and mobile screen sizes

There are other places where FeatureExample component is used so would be good to hunt those down and check that nothing breaks.

For example in Activity log:

<img width="904" alt="Screenshot 2019-05-13 at 16 59 18" src="https://user-images.githubusercontent.com/87168/57627644-0aa0b700-75a1-11e9-98fb-9277609cbb4b.png">
